javascript/index.html

```diff
@@ -57,7 +57,7 @@ <h1 class="page-header">
     </p>
      
     <h3>History</h3>
-    <p>This course was given by <a href="http://www.pamelafox.org">Pamela Fox</a> in May, 2012 as part of <a href="www.meetup.com/Girl-Develop-It-San-Francisco/">GirlDevelopIt</a> in San Francisco, California. The course was delivered over two weeks, on subsequent Monday and Wednesday evenings, 6-8pm.
+    <p>This course was given by <a href="http://www.pamelafox.org">Pamela Fox</a> in May, 2012 as part of <a href="www.meetup.com/Girl-Develop-It-San-Francisco/">GirlDevelopIt</a> in San Francisco, California. More recently, this has been taught as part of a multi day weekend workshop by Brenda Jin, Bianca Gandolfo, and Claire Bendersky.
```

Comment:
We might also want to add the day breakdowns so that they match our Post in channel